assets/js/react/GamePlayer/components/DoroConsole.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react"
+import PropTypes from "proptypes"
+
+import DoroOutput from "./DoroOutput"
+
+class DoroConsole extends Component {
+  static propTypes = {
+    player: PropTypes.object.isRequired,
+    socket: PropTypes.object.isRequired,
+  }
+
+  constructor(props) {
+    super(props)
+
+    const { player, socket } = props
+    const channel = socket.channel(`player:${player.playerId}`)
```

Comment:
this construct feels a little weird.  There is probably a better way to share the socket/channel connection.  but i want to talk about it before i make another mod.  This is currently working.